### DISCUSSION TITLE
Deploying to Vercel leads to an deployment error

### DISCUSSION THREAD

**Discussion by rjmolina13** [spencerwooo/onedrive-vercel-index#374] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/38134522/152907288-0f4348ec-89f7-412c-b534-c6178ba1973f.png)
![image](https://user-images.githubusercontent.com/38134522/152907330-e1f0303b-7401-484d-9b4a-ca1fdbfa6bdc.png)

This is after forking this repo and following steps at [Getting Started](https://onedrive-vercel-index.spencerwoo.com/docs/getting-started).

**Comment by myl7** [comment]
The latest version of the app should be using `pnpm` other than `yarn`. If you are using an old version, you may try updating your fork to the latest commit and then check if the problem still exists.

**Comment by BrandonStudio** [comment]
Default settings worked yesterday, but don't work today......

**Comment by spencerwooo** [comment, marked as answer]
Does manually changing build commands to `pnpm build` and install commands to `pnpm install` work?

https://onedrive-vercel-index.spencerwoo.com/docs/getting-started#import-and-deploy-in-vercel

**Reply by rjmolina13** [reply]
@spencerwooo @myl7 Overriding the build and install commands from the project settings, did fix it. Thank you for the help and sorry for the inconvenience. Also after this, I got the 504 error, I then proceeded by reading the docs, fixed integration with Upstream, re-oauthorized and re-deployed. Boom! Got it working! Cheers 🍻

**Reply by spencerwooo** [reply]
Glad to hear you got it working. Cheers 🥂

**Reply by rjmolina13** [reply]
@spencerwooo I have another problem I'm hoping you could assist me with this therefore would like to start another discussion, would it be okay if we continue discussing it here or should I start another discussion?
The problems are being: 

- .mkv aren't playing properly / doesn't load
![image](https://user-images.githubusercontent.com/38134522/153001801-eeeeacd1-7e3f-4a45-89a6-04e6d79e4219.png)

- ~~hosted icons/,pngs on forked github repo converted from https://favicon.io/favicon-converter/ shows a broken image on the navigation bar. (tried different sizes and redeployed multiple times to no avail.)~~
![image](https://user-images.githubusercontent.com/38134522/153001760-4ad0bc3d-0d29-4252-9639-7ea8f8b3d3a2.png)
**Scratch that, after redeploying a bunch of times and giving itself time. The image just successfully loaded.**

PS. Site is: https://onedrive-vercel-index-five-woad.vercel.app/

**Reply by spencerwooo** [reply]
Regarding your first issue - Is your MKV video in [HEVC](https://en.wikipedia.org/wiki/High_Efficiency_Video_Coding) format? If so, [browser support for HEVC/H.265 videos](https://caniuse.com/hevc) is extremely limited.
